Scripts/SDL/Kveer.sdl

```diff
@@ -59,3 +59,16 @@ VAR BOOL    kverMystLinkingBookVis[1]    DEFAULT=0 DEFAULTOPTION=VAULT
 ## Reward Clothing
 
 }
+
+STATEDESC Kveer
+{
+	VERSION 2
+
+    # Boolean variables
+    VAR BOOL    kverMystLinkingBookVis[1]    DEFAULT=0 DEFAULTOPTION=VAULT
+
+    ## Reward Clothing
+    VAR BOOL    kverAtrusVis[1]              DEFAULT=0 DEFAULTOPTION=VAULT
+    VAR BOOL    kverAtrusGet[1]              DEFAULT=1
```

Comment:
I seem to remember that this variable is no longer needed...? Should this PR be closed?

Reply:
Sorry I'm only just now getting back to this (finally got around to cleaning up git branches.

This is exactly how it now appears now in MOULa. In hindsight, probably should have adjusted the takable clothing script node tree, but I didn't know that was a thing until after this deed was done. 😊